src/pages/HomePage/HomePage.js

```diff
@@ -23,10 +30,90 @@ const link = css`
 class HomePage extends Component {
 	state = {
 		showModal: false, // modal for create new app
+		appName: '',
```

Comment:
Can you instead create a separate component for CreateAppModal? That will be good for various reasons:

- We will need create-new-app functionality in the in-app views (header) as well
- It will keep things modular.